### PR TITLE
Add Jackpot Command & !gamble timeout

### PR DESCRIPTION
**Pull request by H4nSolo**
Added Things:

[] Timeout Function to set up generally Gamble timeout
[] Jackpot Command to the current Jackpot Value (if Jackpot is enabled)

Changes: 
[] -

###### CHECKLIST

- [ ] tests are changed or added
- [ ] documentation is changed or added
- [ ] locales are changed or added
- [ ] db relationship (tools/database) are changed or added
- [ ] commit message follows [commit guidelines](https://github.com/sogehige/sogeBot/blob/master/CONTRIBUTING.md#commit-guidelines)

### CODE REVIEW

File: src/bot/games/gamble.ts [hunk 1]
```diff
@@ -37,96 +37,150 @@ class Gamble extends Game {
...
+
+    if (this.enableJackpot) {
+      const currentPointsOfUser = await pointsSystem.getPointsOf(opts.sender.userId);
+      message = prepare('gambling.gamble.jackpotinfo', {
+        pointsName: await pointsSystem.getPointsName(currentPointsOfUser),
+        jackpotName: await pointsSystem.getPointsName(this.jackpotValue),
+        jackpot: Math.ceil(this.jackpotValue),
+      });
+
+      /*message = prepare('gambling.gamble.winJackpot', {
```
Comment (sogehige) [inline src/bot/games/gamble.ts:57]: Why it is commented and what is difference between winJackpot and jackpotinfo?

File: src/bot/games/gamble.ts [hunk 2]
```diff
@@ -37,96 +37,150 @@ class Gamble extends Game {
...
   async main (opts: CommandOptions): Promise<CommandResponse[]> {
     let points, message;
 
     opts.sender['message-type'] = 'chat'; // force responses to chat
-    try {
-      const parsed = opts.parameters.trim().match(/^([\d]+|all)$/);
-      if (_.isNil(parsed)) {
-        throw Error(ERROR_NOT_ENOUGH_OPTIONS);
+    //mindforger timeout check
+    var has_timeout = 0;
```
Comment (sogehige) [inline src/bot/games/gamble.ts:77]: don't use `var`...

File: src/bot/games/gamble.ts [hunk 3]
```diff
@@ -37,96 +37,150 @@ class Gamble extends Game {
   chanceToTriggerJackpot = 5;
   @shared(true)
   jackpotValue = 0;
+  @settings("timeout")
+  gambletimeout = 30
```
Comment (sogehige) [inline src/bot/games/gamble.ts:41]: please use properly camelCase

File: src/bot/games/gamble.ts [hunk 4]
```diff
@@ -37,96 +37,150 @@ class Gamble extends Game {
...
+
+  timeoutList: Dictionary<number> = {};
```
Comment (sogehige) [inline src/bot/games/gamble.ts:43]: you can use `this.timeouts` from `_interface`

File: src/bot/games/gamble.ts (hunk 2 above)
Comment (sogehige) [inline src/bot/games/gamble.ts:77]: use camelCase

File: src/bot/games/gamble.ts (hunk 1 above)
Comment (H4nSolo) [reply src/bot/games/gamble.ts:57]: The Commented line was only for me Temporaly an can be deletet ^^
i have forget them to delete... sry

**Pull request closed without merging**